#DualShock4 Hat State message
int8 hat_x
int8 hat_y
